packages/demand-bloc-payment/src/types.ts

```diff
@@ -42,7 +48,12 @@ export type Provider = {
   tokenizeHostedFields(): Promise<TokenizePayload>
   validatePaymentForm(): boolean
   completeThreeDSecureVerification(params?: CompleteThreeDSecureVerificationParams): Promise<string | Error>
-  startThreeDSecureVerification(amount: number, nonce: string): Promise<string | Error>
+  startThreeDSecureVerification(
+    amount: number,
+    nonce: string,
+    bin?: HostedFieldsAccountDetails,
```

Comment:
Same as above, we can use `any` type for now.

Reply:
tutaj zrobiłem błąd, to jest string